runtime/vm/vmprops.c

```diff
@@ -779,6 +779,20 @@ initializeSystemProperties(J9JavaVM * vm)
 		goto fail;
 	}
 
+#if defined(J9JDK_EXT_NAME)
+	rc = addSystemProperty(vm, "jdk.extensions.name", J9JDK_EXT_NAME, 0);
+	if (J9SYSPROP_ERROR_NONE != rc) {
+		goto fail;
+	}
+#endif
+
+	#if defined(J9JDK_EXT_VERSION)
```

Comment:
nitpick: Remove the indent on this line.

Reply:
Formatting fixed